design-documents/graph-ql/coverage/CartPromotions.graphqls

```diff
@@ -0,0 +1,53 @@
+type Cart{
+
+    /// Deprecated Feilds
+    applied_coupon: applied_coupon @deprecated(reason: "Use CartPromotions.coupons instead ")                               
+    applied_gift_cards: [AppliedGiftCard] @deprecated(reason: "Use CartPrices.gift_cards instead ") 
```

Comment:
If we want to move applied gift cards and store credit, to me it would be more logical to see those as part of selected payment methods, not part of cart prices.

Applied coupons seem to be good as is. What is the logic behind moving them to prices? None of these are actually prices.

We may think about different name for `prices` and `CartPrices` which would be generic enough to contain all of these fields.